tests/lib/nested.sh

```diff
@@ -744,6 +744,10 @@ defaults:
     journal:
       persistent: true
 EOF
+                        # add snapd debug and log to serial console for extra
+                        # visibility what happens when a machine fails to boot
+                        echo 'console=ttyS0 snapd.debug=1 systemd.journald.forward_to_console=1 ' > pc-gadget/cmdline.extra
```

Comment:
This is breaking some tests because it is sending some DEBUG messages to the stderr and the test 
external:ubuntu-core-20-64:tests/smoke/install is checking the stderr is empty.